src/objects.c

```diff
@@ -1274,7 +1289,7 @@ P11PROV_OBJ *p11prov_obj_find_associated(P11PROV_OBJ *obj,
     slotid = p11prov_obj_get_slotid(obj);
 
     ret = p11prov_get_session(obj->ctx, &slotid, NULL, NULL,
-                              CK_UNAVAILABLE_INFORMATION, NULL, NULL, true,
+                              CK_UNAVAILABLE_INFORMATION, NULL, NULL, false,
```

Comment:
So the "fix" boils down to not logging in?
This may work in some cases if you are already logged in for some reason, but will fail in cases where no previous login was attempted.
I think we need a test cases that carefully manages how login is done and tries a public key + find associated and shows how gracefully (or not) this fails.

Reply:
On second thought this might be sufficient.

Because most of the cases of finding associated keys go from private to public or certificate to public anyway, therefore either you are already logged in or probably do not need to be.

Probably ok for now.